module/PowerShellEditorServices/PowerShellEditorServices.psd1

```diff
@@ -12,7 +12,7 @@
 RootModule = 'PowerShellEditorServices.psm1'
 
 # Version number of this module.
-ModuleVersion = '1.10.3'
+ModuleVersion = '1.11.0'
```

Comment:
This will get bumped to `1.11.1` before the release, right?

Reply:
Yeah it will be 1.12.0 because weve got features!

Reply:
Even better.